ios/swiftlint.md

```diff
@@ -12,26 +12,27 @@ Linting is important and we want to adapt a uniform linter across all iOS projec
 
 ### SwiftLint Installation
 
-To use swiftlint, we just have to install it with brew. In terminal, run:
+To use SwiftLint, we just have to install it with brew. In terminal, run:
 
 ```text
 brew install swiftlint
 ```
 
-
+Then, add SwiftLint to your Podfile
+```text
+pod 'SwiftLint'
```

Comment:
we aren't using swiftlint via a pod. We discussed this privately and decided to stick with brew